packages/ibm-products/src/components/Coachmark/Coachmark.stories.jsx

```diff
@@ -94,7 +98,7 @@ const Template = (args) => {
   });
   const theme = getSelectedCarbonTheme();
   const content = (
-    <Coachmark {...args} theme={theme}>
+    <Coachmark closeIconDescription="Close" {...args} theme={theme}>
```

Comment:
if `closeIconDescription` is coming from `args` we don't need to declare it here